src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -18,7 +18,6 @@ public abstract class HttpClientHandlerTest_Http2 : HttpClientHandlerTestBase
 
         public static bool SupportsAlpn => PlatformDetection.SupportsAlpn;
 
-        [ConditionalFact(nameof(SupportsAlpn))]
```

Comment:
These tests are no longer going to run because you removed ConditionalFact but didn't replace it with Fact.  I'm also not clear why these will pass now, given that we're not setting that switch or environment variable.

Reply:
fixed. I thought I missed something but since the test did not run, I missed that.